### ISSUE TITLE
Drag and drop - Talkback and iPad bugs

### ISSUE THREAD

**Issue by LFDanLu** [adobe/react-spectrum#1730]
TODO: retest and see what is still valid

# 🐛 Bug Report 
Two investigation items from testing session:

- iPad Safari - dragged element has white corners (DG) (OS specific can't fix) (7/26/22 verified DL)
- iPad Safari greys out the dragged element text (DG) (browser bug, find in wiki) (7/26/22 verified DL, browser bug, confirmed that it is iOS Safari specific, doesn't happen in desktop Safari) https://bugs.webkit.org/show_bug.cgi?id=223595
- ~~In dialog, open the dialog drag the item and drop it outside, try to close the dialog by pressing the dismiss button, it doesn't close on the first click. It sounds like a safari bug~~ Cannot reproduce anymore (7/26/22 DL)
- ~~Talkback exposes a focusable grid element (?) that on double tap will select the item and render a orange box around it (DL)~~ Cannot reproduce in DnD stories anymore (7/26/22 DL)

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |
| Browser          |
| Operating System |

**Comment by matthewdeutsch** [comment]
browser/OS bug, closing out
